book_storage.rb

```diff
@@ -0,0 +1,29 @@
+require_relative('./book')
+class BookStorage
+  def initialize
+    @books = []
+  end
+
+  def create_book
+    print 'Title: '
+    book_title = gets.chomp
+
+    print 'Author: '
+    book_author = gets.chomp
+
+    puts 'Book created successfully!'
+
+    book = Book.new(book_title, book_author)
+    @books.push({
+                  output: "Title: #{book.title}, Author: #{book.author}",
+                  object: book
+                })
+  end
```

Comment:
- [ ] in this method you are still printing to the console, gathering information and processing data, which is not in line with the single responsibility rule as explained in the previous review. please split the method in more manageable pieces. please take the first comment as an example of how to fix this.